mariadb_kernel/client_config.py

```diff
@@ -122,6 +122,12 @@ def get_server_paths(self):
             os.path.dirname(self.default_config["server_pid"]),
         ]
 
+    def get_server_pidfile(self):
+        return self.default_config["server_pid"]
+
+    def get_server_socketfile(self):
```

Comment:
True, I implemented some cleanup code for the lingering socket/PID  directories but I removed that code as it could potentially be dangerous. 

Will delete.